src/components/EventCard/EventCard.js

```diff
@@ -0,0 +1,95 @@
+                            </svg>
+                            <p id={styles.para}>{time}</p>
+                        </div>
+                        <div className={styles.iconText}>
+                            <svg xmlns="http://www.w3.org/2000/svg" width="21" height="21" viewBox="0 0 24 24">
+                                <path fill="#CBCBCB" d="M12 12q.825 0 1.413-.588T14 10q0-.825-.588-1.413T12 8q-.825 0-1.413.588T10 10q0 .825.588 1.413T12 12Zm0 7.35q3.05-2.8 4.525-5.088T18 10.2q0-2.725-1.738-4.462T12 4Q9.475 4 7.737 5.738T6 10.2q0 1.775 1.475 4.063T12 19.35ZM12 22q-4.025-3.425-6.012-6.362T4 10.2q0-3.75 2.413-5.975T12 2q3.175 0 5.588 2.225T20 10.2q0 2.5-1.988 5.438T12 22Zm0-11.8Z"/>
+                            </svg>
+                            <p id={styles.para}>{location}</p>
+                        </div>
+                        <div className={styles.iconText}>
+                            <svg xmlns="http://www.w3.org/2000/svg" width="21" height="21" viewBox="0 0 16 16">
+                                <path fill="#cbcbcb" d="M2 12.5a.5.5 0 0 1 .5-.5h7a.5.5 0 0 1 0 1h-7a.5.5 0 0 1-.5-.5zm0-3a.5.5 0 0 1 .5-.5h11a.5.5 0 0 1 0 1h-11a.5.5 0 0 1-.5-.5zm0-3a.5.5 0 0 1 .5-.5h11a.5.5 0 0 1 0 1h-11a.5.5 0 0 1-.5-.5zm4-3a.5.5 0 0 1 .5-.5h7a.5.5 0 0 1 0 1h-7a.5.5 0 0 1-.5-.5z"/>
+                            </svg>
+                            <p id={styles.para}>{description}</p>
+                        </div>
+                    </div>
+                </div>
+                <div className={styles.bottom}>
+                    {badge}
+                    <Button className={styles.button}>
+                        <p id={styles.buttonText}>{buttText}</p>
+                    </Button>
+                </div>
+            </div>
+        </div>
+        
+    )
+}
+
+function OwnerBadge() {
+    return (
+        <div className={styles.iconText}>
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+                <path fill="#7aa3ff" d="m12 23l-3-3H5q-.825 0-1.413-.588T3 18V4q0-.825.588-1.413T5 2h14q.825 0 1.413.588T21 4v14q0 .825-.588 1.413T19 20h-4l-3 3Zm0-11q1.45 0 2.475-1.025T15.5 8.5q0-1.45-1.025-2.475T12 5q-1.45 0-2.475 1.025T8.5 8.5q0 1.45 1.025 2.475T12 12Zm0 8.2l2.2-2.2H19v-1.15q-1.35-1.325-3.138-2.087T12 14q-2.075 0-3.863.763T5 16.85V18h4.8l2.2 2.2Z"/>
+            </svg>
+            <p id={styles.badgeText}>Your Group</p>
+        </div>
+    )
+}
```

Comment:
generally it's syntactic practice to have secondary components above the export default component, so move it up

Reply:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/112282141/227413942-7e887754-0549-4b83-9968-54c45ce4a215.png">

Moved